src/components/tooltip/tooltip.js

```diff
@@ -175,8 +175,9 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
     }
 
     function showTooltip() {
-      // Insert the element before positioning it, so we can get the position
+      // Insert the element and position at top left, so we can get the position
       // and check if we should display it
+      element.css({top: 0, left: 0});
```

Comment:
What is the action item right now? I'd like to fix this bug but it isn't clear to me what steps I should be taking. Thanks!